tests/tests_unit/rules/test_models/test_information_rules.py

```diff
@@ -482,3 +482,50 @@ def test_rdf_properties(self, raw: InformationInputProperty):
         prop = InformationProperty.model_validate(raw.dump(default_prefix="power"))
 
         assert isinstance(prop, InformationProperty)
+
+    @pytest.mark.parametrize(
+        "raw, expected",
+        [
+            pytest.param(
+                InformationInputProperty(
+                    "cdf_cdm:CogniteAsset(version=v1)",
+                    "name",
+                    "text",
+                ),
+                ClassEntity(prefix="cdf_cdm", suffix="CogniteAsset", version="v1"),
+                id="CogniteAsset name",
+            )
+        ],
+    )
+    def test_validate_class_entity(self, raw: InformationInputProperty, expected: ClassEntity) -> None:
+        prop = InformationProperty.model_validate(raw.dump(default_prefix="my_space"))
+
+        assert prop.class_ == expected
+
+
+class TestInformationClass:
+    @pytest.mark.parametrize(
+        "raw, class_, implements",
+        [
+            pytest.param(
```

Comment:
I don't see any meta activities here - the pytest.param() call isn't needed (it isn't harmful either, but just some noise)

Reply:
You are correct, removed it.